.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,29 @@
+name: Nightly
+on:
+  schedule:
+    - cron: '30 12 * * *'
```

Comment:
Can you comment what time this would be? Is it 12:30 GMT? Do we want to run this on weekends too? Will need to make sure we're on top of failures on the weekend if they arise.

Reply:
This is a well known cron tab format, you can use things like https://crontab.guru/ to translate. Not sure about the weekend, but technically since all merges pass build this should never happen. And if something happens it's better to have nightlies for the other days. Since we may merge code during the weekend it's important to have them build every day.